pyvista/core/filters/data_set.py

```diff
@@ -4613,6 +4613,76 @@ def shrink(self, shrink_factor=1.0, progress_bar=False):
             return output.extract_surface()
         return output
 
+    def tessellate(self, max_n_subdivide=3, merge_points=True, progress_bar=False):
+        """Tessellate a mesh.
+
+        This filter approximates nonlinear FEM-like elements with linear
+        simplices. The output mesh will have geometry and any fields specified
+        as attributes in the input mesh's point data. The attribute's copy
+        flags are honored, except for normals.
+
+        For more details see `vtkTessellatorFilter <https://vtk.org/doc/nightly/html/classvtkTessellatorFilter.html#details>`_.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Parameters
+        ----------
+        max_n_subdivide : int, optional
+            Maximum number of subdivisions.
+            Defaults to ``3``.
+
+        merge_points : bool, optional
+            The adaptive tessellation will output vertices that are not shared among cells,
+            even where they should be. This can be corrected to some extents.
+            Defaults to True.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Returns
+        -------
+        pyvista.DataSet
+            Dataset with tessellated mesh.  Return type matches input.
+
+        Examples
+        --------
+        First, plot the high order FEM-like elements .
+
+        >>> import pyvista
+        >>> import numpy as np
+        >>> points = np.array(
+        ...     [
+        ...         [0.0, 0.0, 0.0],
+        ...         [2.0, 0.0, 0.0],
+        ...         [1.0, 2.0, 0.0],
+        ...         [1.0, 0.5, 0.0],
+        ...         [1.5, 1.5, 0.0],
+        ...         [0.5, 1.5, 0.0],
+        ...     ]
+        ... )
+        >>> cells = np.array([6, 0, 1, 2, 3, 4, 5])
+        >>> cell_types = np.array([69])
+        >>> mesh = pyvista.UnstructuredGrid(cells, cell_types, points)
+        >>> mesh.plot(show_edges=True, line_width=5)
+
+        Now, plot the tessellated mesh.
+
+        >>> tessellated = mesh.tessellate()
+        >>> tessellated.clear_data()  # cleans up plot
+        >>> tessellated.plot(show_edges=True, line_width=5)
+
+        """
+        alg = _vtk.vtkTessellatorFilter()
+        alg.SetInputData(self)
+        alg.SetMergePoints(merge_points)
+        alg.SetMaximumNumberOfSubdivisions(max_n_subdivide)
+        _update_alg(alg, progress_bar, 'Tessellating Mesh')
+        output = pyvista.wrap(alg.GetOutput())
```

Comment:
Does the usual `_get_output(alg)` not work here?

Reply:
Oh. I didn't know it. Thanks :)
```suggestion
        output = _get_output(alg)
```